users/autostart.rst

```diff
@@ -20,32 +20,111 @@ there are a number of easy solutions.
 Task Scheduler
 ~~~~~~~~~~~~~~
 
-#. Start the `Task Scheduler <https://en.wikipedia.org/wiki/Windows_Task_Scheduler>`__ (``taskschd.msc``).
-#. Create a New Task ("Action" menu -> "Create Task...").
-#. General Tab:
-    #. Name the task (for example 'Syncthing').
-    #. Check "Run whether user is logged on or not".
-    #. (recommended) Check "Do not store password. The task will only have access to local resources". Leave this option unchecked if you intend to store your folders on network drives and such.
-#. Triggers Tab:
-    #. Click "New...".
-    #. Set "Begin the task" to "At Startup".
-    #. (optional) Choose a delay.
-    #. Make sure Enabled is checked.
-    #. Click "OK".
-#. Actions Tab:
-    #. Click "New...".
-    #. [Action] should be set as "Start a program".
-    #. Enter the path to syncthing.exe in "Program/Script".
-    #. (optional) Enter the path to the parent folder of syncthing.exe in "Start in (optional)". This will allow you to use paths relative to this folder in Syncthing.
-    #. Click "OK".
+Task Scheduler is a built-in administrative tool, which can be used to
+start Syncthing automatically either at user log on, or at system
```

Comment:
Just a tiny nit: To my understanding there's the verb "log in" when you enter credentials to get access to something, and the noun "login" describing that process. So to my understanding it should be "login" here instead of "log on" (and same for all other occurrences). Disclaimer: I am not a language authority, so might be wrong - refute me if appropriate ;)

Reply:
I actually explained it at the very end of the commit message :wink:.

>Fix the language to match the terminology used by Microsoft (e.g. log
in -> log on).

The key here is that Microsoft uses "log on" (with a space), both as a verb, and a noun. Specifically in Task Scheduler, it clearly says `at log on`, so I think that it is better to stick to the Microsoft's terminology to avoid any confusion, at least in the Windows-related instructions.

I think that these are related:

https://stackoverflow.com/questions/406016/ui-terminology-logon-vs-login
https://stackoverflow.com/a/406522 (regarding Microsoft)